### PR TITLE
[Serverless] Update feeder/witness to use standalone distribute command

### PR DESCRIPTION
**Pull request by AlCutter**

### CODE REVIEW

**Review by mhutchinson — APPROVED** [review]

File: serverless/cmd/distribute/github/main.go [hunk 1]
```diff
@@ -134,6 +134,16 @@ func distributeOnce(ctx context.Context, opts *options, repo github.Repository)
 		return fmt.Errorf("checkpoint has %d witness sigs, want %d", nWitSigs, want)
 	}
 
+	logDir := filepath.Join(opts.distributorPath, "logs", logID)
+	found, err := alreadyPresent(witnessNote.Text, logDir, repo, opts.witSigV)
+	if err != nil {
+		return fmt.Errorf("couldn't determind whether to distribute: %v", err)
```
Comment (mhutchinson) [inline serverless/cmd/distribute/github/main.go:140]:
```suggestion
		return fmt.Errorf("couldn't determine whether to distribute: %v", err)
```

File: serverless/cmd/distribute/github/main.go [hunk 2]
```diff
@@ -164,6 +173,33 @@ func distributeOnce(ctx context.Context, opts *options, repo github.Repository)
 	return repo.CreatePR(ctx, fmt.Sprintf("Witness %s@%d", opts.witSigV.Name(), wCp.Size), branchName)
 }
 
+func alreadyPresent(cpBody string, logDir string, repo github.Repository, wSigV note.Verifier) (bool, error) {
```
Comment (mhutchinson) [inline serverless/cmd/distribute/github/main.go:176]: Maybe alreadyIntegrated would be clearer? This isn't just checking that we have generated this checkpoint before, but that it's been merged into the distributor.

**Review by mhutchinson — APPROVED** [review]

File: serverless/cmd/distribute/github/main.go (hunk 1 above)
Comment (AlCutter) [reply serverless/cmd/distribute/github/main.go:140]: ta, done

File: serverless/cmd/distribute/github/main.go (hunk 2 above)
Comment (AlCutter) [reply serverless/cmd/distribute/github/main.go:176]: done

**Pull request merged**